.github/workflows/twister_dfu_nrf52840dk.yml

```diff
@@ -125,6 +125,8 @@ jobs:
           PATH=$PATH:/opt/nrf-command-line-tools/bin
           nrfjprog -i
           golioth releases rollback '*'
+          # FIXME: workaround when missing the initial IDLE log
+          sed -i '/LOG_DBG("Start DFU sample");/a \\tk_sleep(K_SECONDS(10));' modules/lib/golioth/samples/dfu/src/main.c
```

Comment:
How about adding `CONFIG_BOOT_DELAY=10000`? Would that help as well? We could put that into `sample.yaml` if it would work.

Reply:
Yes! I didn't know about this symbol, it's a much better way to handle it. Thanks.